### ISSUE TITLE
Update query used for crmp.climo_obs_count_mv

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/pycds#159]
At present:
- This "matview" is a table whose contents are manually managed to give equivalent-to-matview effects. It is not managed at present by PyCDS.
- Its contents are determined by the following view:
  ```
  CREATE OR REPLACE VIEW crmp.climo_obs_count_v
   AS
   SELECT count(*) AS count,
      obs_raw.history_id
     FROM obs_raw
       JOIN meta_vars USING (vars_id)
    WHERE meta_vars.cell_method::text ~ '(within|over)'::text
    GROUP BY obs_raw.history_id;
  ```
- The query defined for the view is incorrect. A more correct and useful query is:
  ```
  SELECT count(*) AS count,
    obs_raw.history_id
   FROM obs_raw
     JOIN meta_vars USING (vars_id)
  WHERE meta_vars.display_name::text ~ 'Climatology'::text
  GROUP BY obs_raw.history_id;
  ``` 

Depending on the phase in which this issue is addressed, the fix may include a (highly desirable) conversion of the table to a genuine native matview, under management (via Alembic) by PyCDS. If not, then the _view_ should be brought under management of PyCDS. In any case, it is desirable to stop ad-hoc management of this matview.

**Comment by rod-glover** [comment]
See also https://redmine.pcic.uvic.ca/issues/1921 for a temporary expedient.

**Comment by rod-glover** [comment]
Addressed in PRs #167 and #171
